crates/cli/src/commands/apply_pattern.rs

```diff
@@ -109,6 +109,8 @@ pub struct ApplyPatternArgs {
     /// Clear cache before running apply
     #[clap(long = "refresh-cache", conflicts_with = "cache")]
     pub refresh_cache: bool,
+    #[clap(long = "language", alias="lang")]
+    pub language: Option<String>,
```

Comment:
Use an enum, so documentation generation and clap will work better.

Reply:
Only certain values are valid here, so you should use an enum.

Reply:
You can use this existing enum: https://github.com/getgrit/gritql/blob/51863c285b717f58a9138390986246082d5d41c4/crates/language/src/target_language.rs#L46